script/controller/tasklist-front.js

```diff
@@ -89,4 +103,57 @@ angular.module('tasklist-front', ['tasklist-back', 'utils'])
         return function (date) {
             return date == null ? "" : $filter('date')(date, 'MMM dd yyyy - HH:mm:ss');
         };
+    })
+
+    .filter('tasksOrder', function () {
+        return function (items) {
+
+            var _reopened = [];
+            var _new = [];
+            var _finished = [];
+
+            angular.forEach(items, function (item) {
+                switch (item.task.status) {
+                    case Status.REOPENED :
+                        _reopened.push(item);
+                        break;
+                    case Status.NEW :
+                        _new.push(item);
+                        break;
+                    case Status.FINISHED :
+                        _finished.push(item);
+                        break;
+                }
+            });
+
+            return _reopened.concat(_new, _finished);
+        }
+    })
```

Comment:
Any sorting by date? A better solution is to use .sort() with proper compare function.